### PR TITLE
content refactor

### PR DESCRIPTION
**Pull request by bwalsh**
@kellrott @adamstruck can you review per today's conversation?
addresses #31
https://bmeg-staging.ddns.net/analyze/getting_started/

### CODE REVIEW

**Comment by kellrott** [comment]
In the python example, can we get the bash and python code into different paragraphs (separated grey backgrounds)?

**Comment by bwalsh** [comment]
@kellrott  updated

File: layouts/partials/navbar.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 <ul class="sidenav" id="mobile">
   <p>Explore</p>
-  <li><a href="/explore">Data Sources</a></li>
+  <li><a href="/explore/data">Data Sources</a></li>
   <li><a href="/building/demo-introduction">Access</a></li>
   <p>Analyse</p`>
   <li><a href="/building/schema">Schema</a></li>
```
[the remark this answers is not in the document]
Comment (adamstruck) [reply layouts/partials/navbar.html]: Actually, it looks like there are a number of outdated links in this mobile sidebar

File: layouts/partials/navbar.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 <ul class="sidenav" id="mobile">
   <p>Explore</p>
-  <li><a href="/explore">Data Sources</a></li>
+  <li><a href="/explore/data">Data Sources</a></li>
   <li><a href="/building/demo-introduction">Access</a></li>
```
[the remark this answers is not in the document]
Comment (bwalsh) [reply layouts/partials/navbar.html]: fixed broken links.  deployed on https://bmeg-staging.ddns.net/

**Pull request merged**